sceptre/sageit/config/prod/ghasvcaccount.yaml

```diff
@@ -0,0 +1,8 @@
+template_path: remote/service-account.yaml
+stack_name: gha-svc-account
+parameters:
+  ServiceUserPolicies:
+    ["arn:aws:iam::797640923903:role/accounts-AWSIAMPortalDeployerRole-17FJXSP3DEYFM"]
```

Comment:
i think you either need to move this on same line as `ServiceUserPolicies` or keep it here and change it to..

```
  ServiceUserPolicies:
    - "arn:aws:iam::797640923903:role/accounts-AWSIAMPortalDeployerRole-17FJXSP3DEYFM"
```

also you cannot pass in the same role that you are creating in the same template

Reply:
Fixed